stix_shifter/src/stix_pattern_parser/stix_pattern_parser.py

```diff
@@ -20,8 +20,7 @@ class PatternTranslator:
         # Treat AND's as OR's -- Unsure how two ObsExps wouldn't cancel each other out.
         ObservationOperators.And: 'OR',
         ObservationOperators.FollowedBy: 'FOLLOWEDBY',
-        ComparisonComparators.IsSubSet: 'ISSUBSET',
-        ComparisonComparators.IsSuperSet: 'ISSUPERSET',
```

Comment:
Same as the above comment. Best to leave SuperSet operator in even though nothing is calling it right now.